--- origsrc/lua_cliargs-820e2d2e3bbc9e8cce3449b7d74330213995052e/src/cliargs/config_loader.lua	2018-04-02 17:14:47.000000000 +0900
+++ src/lua_cliargs-820e2d2e3bbc9e8cce3449b7d74330213995052e/src/cliargs/config_loader.lua
@@ -120,7 +120,7 @@ return {
   --- [1] http://doc.lubyk.org/yaml.html
   from_yaml = function(filepath)
     local src, config, err
-    local yaml = require 'yaml'
+    local lyaml = require 'lyaml'
 
     src, err = read_file(filepath)
 
@@ -128,7 +128,7 @@ return {
       return nil, err
     end
 
-    config, err = yaml.load(src)
+    config, err = lyaml.load(src)
 
     if not config and err then
       return nil, err
